src/app/containers/Home/Home.js

```diff
@@ -53,45 +57,45 @@ class Home extends Component {
     let page = 1 // TODO add 'more' feature to list more txs
 
     this.setState({ transactionHistoryError: '' }, () => {
-      Neoscan.getAddressAbstracts(account.address, page)
-        .then(results => {
-          if (results && results.data) {
-            let totalPages = results.data.total_pages
-            let pageSize = results.data.page_size
-            let pageNumber = results.data.page_number
-            let totalTxs = results.data.total_entries
-            let txs = {}
-            txs.address = account.address
-            txs.data = []
-            txs.total = totalTxs
-            txs.viewing = 'Viewing ' + pageSize + ' of ' + totalTxs + ' transactions on Page ' + pageNumber + ' of ' + totalPages
-
-            if (results.data && results.data.entries) {
-              accountActions.setTransactions({})
-              return results.data.entries.map(tx => {
-                return Neoscan.getTransaction(tx.txid).then(txDetail => {
-                  txDetail.stringRemarks = []
-
-                  txDetail.attributes.map((remark, i) => {
+      Neoscan.getAddressAbstracts(account.address, page).then(results => {
```

Comment:
this whole block is duplicate code, might by worth optimizing this

Reply:
Wait, what do you mean? The getAddressAbstracts() call retrieves the summary pager data.